lib/tasks/auth_tokens.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+namespace :auth_tokens do
+  desc "Verify each authorized AuthToken that it's still authorized"
+  task :reverify_authorized, %i[count start] => :environment do |_task, args|
+    exit if ENV["READ_ONLY"].present?
```

Comment:
When is `READ_ONLY` set?

Reply:
This is a good question! The application seems to (or did) support some kind of [read only mode](https://github.com/librariesio/libraries.io/blob/268d5397c91d8c598b2af9fad0d9dd972bb4ed64/app/controllers/application_controller.rb#L281-L283) but it does not seem to be documented, nor referenced in any of our infra. Happy to stop cargo culting this line if we don't know we need it